src/shared_utilities/drivers.jl

```diff
@@ -1480,27 +1480,34 @@ function make_update_drivers(d::PrescribedSoilOrganicCarbon{FT}) where {FT}
 end
 
 """
-     prescribed_forcing_era5(era5_ncdata_path,
-                             surface_space,
-                             start_date,
-                             toml_dict::CP.ParamDict,
-                             FT;
-                             gustiness=1,
-                             max_wind_speed = nothing,
-                             c_co2 = TimeVaryingInput((t) -> 4.2e-4),
-                             time_interpolation_method = LinearInterpolation(PeriodicCalendar()),
-                             regridder_type = :InterpolationsRegridder,
-                             interpolation_method = Interpolations.Constant(),)
+    prescribed_forcing_era5(start_date,
+                            stop_date,
+                            use_lowres_forcing,
```

Comment:
maybe this should be a kwarg, with a value of false by default? and the interpolation method should be linear by default.

(the low res forcing/ constant interp is the non-default case)